examples/google-alloydb-vectorstore-example/alloydb_vectorstore_example.go

```diff
@@ -177,19 +172,10 @@ func main() {
 		log.Fatal(err)
 	}
 
-	docs, err := vs.SimilaritySearch(ctx, "Japan")
+	docs, err := vs.SimilaritySearch(ctx, "Japan", 0)
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	fmt.Println("Docs:", docs)
```

Comment:
I want us to have a filtered example. Currently we only support strings

Reply:
done!